aiidalab_widgets_base/crystal_sim_crystal.py

```diff
@@ -0,0 +1,55 @@
+import itertools
+import numpy as np
+import spglib
+
+from ase.spacegroup import crystal
+from ase.data import atomic_numbers, atomic_names
+from ase.spacegroup import Spacegroup
+from numpy.linalg import norm
+
+
+def ase_to_spgcell(ase_atoms):
+    return (ase_atoms.get_cell(),
+            ase_atoms.get_scaled_positions(),
+            ase_atoms.get_atomic_numbers())
+def a_equiv_b(a,b):
```

Comment:
perhaps a more expressive  name might be "check_crystal_equivalence(crystal_a, crystal_b)"